src/guild.h

```diff
@@ -69,6 +69,12 @@ class Guild
 		void setMotd(const std::string& motd) {
 			this->motd = motd;
 		}
+		uint32_t getResidenceId() const {
```

Comment:
Instead of "residence" use "house", it's an actual entity in DB and really it's a house.